backend/services/implementations/__tests__/schoolService.test.ts

```diff
@@ -31,72 +38,48 @@ describe("mongo schoolService", (): void => {
     await db.clear();
   });
 
-  it("create school for valid teachers", async () => {
-    // set up test case
-
-    // set up test users to return from user service
-    const testUsers = [
-      {
-        id: "56cb91bdc3464f14678934ca",
-        firstName: "Teacher",
-        lastName: "One",
-        authId: "123",
-        role: "Admin",
-      },
-      {
-        id: "56cb91bdc3464f14678934cb",
-        firstName: "Teacher",
-        lastName: "Two",
-        authId: "456",
-        role: "Admin",
-      },
-    ];
+  it("getAllSchools", async () => {
+    await SchoolModel.insertMany(testSchools);
+    // mock return value of user service
+    userService.findAllUsersByIds = jest.fn().mockReturnValue(testUsers);
 
-    // set up test school to create
-    const testSchool = {
-      name: "some-name",
-      country: "some-country",
-      subRegion: "some-region",
-      city: "some-city",
-      address: "some-address",
-      teachers: [testUsers[0].id, testUsers[1].id],
-    };
+    const res = await schoolService.getAllSchools();
+
+    res.forEach((school: SchoolResponseDTO, i) => {
+      expect(school.id).not.toBeNull();
+      expect(school.name).toEqual(testSchools[i].name);
+      expect(school.country).toEqual(testSchools[i].country);
+      expect(school.subRegion).toEqual(testSchools[i].subRegion);
+      expect(school.city).toEqual(testSchools[i].city);
+      expect(school.address).toEqual(testSchools[i].address);
+      expect(school.teachers).toEqual(testUsers);
+    });
+  });
 
+  it("create school for valid teachers", async () => {
     // mock return value of user service
     userService.findAllUsersByIds = jest.fn().mockReturnValue(testUsers);
 
     // execute
-    const createdSchool = await schoolService.createSchool(testSchool);
+    const createdSchool = await schoolService.createSchool(testSchools[0]);
 
     // assert
     expect(createdSchool.id).not.toBeNull();
-    expect(createdSchool.name).toEqual(testSchool.name);
-    expect(createdSchool.country).toEqual(testSchool.country);
-    expect(createdSchool.subRegion).toEqual(testSchool.subRegion);
-    expect(createdSchool.city).toEqual(testSchool.city);
-    expect(createdSchool.address).toEqual(testSchool.address);
+    expect(createdSchool.name).toEqual(testSchools[0].name);
+    expect(createdSchool.country).toEqual(testSchools[0].country);
+    expect(createdSchool.subRegion).toEqual(testSchools[0].subRegion);
+    expect(createdSchool.city).toEqual(testSchools[0].city);
+    expect(createdSchool.address).toEqual(testSchools[0].address);
```

Comment:
if you decide to do the above refactor - you can replace this entire block with that function call